core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataIndexAliasesService.java

```diff
@@ -88,12 +88,13 @@ protected ClusterStateUpdateResponse newResponse(boolean acknowledged) {
 
                 @Override
                 public ClusterState execute(ClusterState currentState) {
-                    return innerExecute(currentState, request.actions());
+                    return updateAliasesInClusterState(currentState, request.actions());
                 }
             });
     }
 
-    ClusterState innerExecute(ClusterState currentState, Iterable<AliasAction> actions) {
+    // public for use in tests
```

Comment:
I don't think these comments add anything over the fact that compilation would break if we remove it without adapting corresponding tests, and our IDEs can show us all the call-sites/overrides.